src/App.js

```diff
@@ -45,20 +51,40 @@ class App extends React.Component {
             </Row>
             <Row className='bottomMenu'>
               <Col>
-                <Dynamic />
+                {
+                  this.state.loadingDone ?
+                    <Static
+                      signInTitle={this.state.signInTitle}
+                      welcomeToDynamoTitle={this.state.welcomeToDynamoTitle}
+                      launchTitle={this.state.launchTitle}
+                      showScreenAgainLabel={this.state.showScreenAgainLabel}
+                    /> : <Dynamic />
+                }
               </Col>
             </Row>
           </Col>
           <Col className='p-0' >
-            <img className='screenBackground' src={base64DynamoBackground}></img>
+            <img className='screenBackground' alt='' src={base64DynamoBackground}></img>
           </Col>
         </Row>
       </Container>
     )
   }
 
-  setWelcomeDynamoTitle(title) {
-    this.setState({ welcomeToDynamoTitle: title })
+  setLabels(labels) {
+    this.setState({
+      welcomeToDynamoTitle: labels.welcomeToDynamoTitle,
+      signInTitle: labels.signInTitle,
+      launchTitle: labels.launchTitle,
+      showScreenAgainLabel: labels.showScreenAgainLabel
+    })
+  }
```

Comment:
Nice, can we add comments for this function to reflect the purpose. Please also add that in the future, devs should expand lables object to contain more resources